custom/icds_reports/migrations/sql_templates/database_views/system_usage_report_view.sql

```diff
@@ -26,7 +26,14 @@ CREATE VIEW system_usage_report_view AS
         COALESCE("agg_awc_monthly"."usage_num_cf", 0) AS "usage_num_cf",
         COALESCE("agg_awc_monthly"."usage_num_gmp", 0) AS "usage_num_gmp",
         COALESCE("agg_awc_monthly"."usage_num_due_list_ccs", 0) + COALESCE("agg_awc_monthly"."usage_num_due_list_child_health", 0) AS "usage_num_due_list_ccs_and_child_health",
-        COALESCE("agg_ls"."num_supervisor_launched", 0) AS "num_supervisor_launched"
+        COALESCE("agg_ls"."num_supervisor_launched", 0) AS "num_supervisor_launched",
+        "agg_awc_monthly"."num_launched_states" AS "num_launched_states",
```

Comment:
just pattern matching, but should these also use the `COALESCE`?

Reply:
Very often, I see that some fields doesn't use coalesce, and some uses. In some cases, same fields in different tables uses it in one  and ignore it in another. Any criterion on when we do that?

Reply:
I think  fields which are expected to have nulls in the db but in the report you don't want nulls and want 0s then it makes sense to use `COALESCE`. Talking about these launched indicators added  from agg_awc_monthly are set to 0 by default, we no need to use `COALESCE`

Reply:
Thanks @Rohit25negi